providers/cloudns/api.go

```diff
@@ -71,21 +71,7 @@ type domainRecord struct {
 
 type recordResponse map[string]domainRecord
 
-var allowedTTLValues = []uint32{
```

Comment:
Using the API is good.  It is better than a constant.
Is the API returning any values that are not on this list?  For backwards compatibility it would be good to know.